src/snyk/common/configuration/configuration.ts

```diff
@@ -203,6 +205,15 @@ export class Configuration implements IConfiguration {
   }
 
   async getToken(): Promise<string | undefined> {
+    // We do not want to get token if the extension is running in a remote environment
+    // This is due to the fact that SecretStorage runs independent of the workspace i.e. it runs on the local machine
+    const remoteNames = ['ssh-remote'];
+    const remoteName = vsCodeEnv.getRemoteName();
+    if (remoteName && remoteNames.includes(remoteName)) {
+      Logger.debug(`Token not retrieved. Extension is running remotely via: ${remoteName}`);
+      return;
```

Comment:
We probably want to retrieve the token from the remote config store instead of not retrieving any token at all